woocommerce/woocommerce-gateway-stripe
# Remove Еxpress Checkout section extra spacing

## Changes proposed in this Pull Request:
Removes extra spacing on the Express Checkout section by mistake.

<img width="770" alt="Screenshot 2022-11-01 at 13 48 38" src="https://user-images.githubusercontent.com/8667118/199307422-b540711d-8cbc-499a-828a-134a3658bec4.png">

## Testing instructions
- Navigate to `WooCommerce` -> `Settings` -> `Payments` -> `Stripe`.
- Verify there is no extra spacing added to the Express Checkout section.

---


**Post merge**
